packages/components/src/core/styles/common/mixins/a11y.ts

```diff
@@ -0,0 +1,8 @@
+export function focusVisibleA11yStyle() {
+  return `
+      &.Mui-focusVisible, &:focus-visible {
```

Comment:
Good idea 👍 

If you wanted to, you could use this util in the Tab component I updated this morning: https://github.com/chanzuckerberg/sci-components/blob/eb6870f6d8bfe213555a5e0c0040308833510ab9/packages/components/src/core/Tabs/style.tsx#L97-L100

Or I can do it later